integration/api/src/main/java/com/cerner/beadledom/integration/api/model/HelloWorldDto.java

```diff
@@ -0,0 +1,49 @@
+package com.cerner.beadledom.integration.api.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+/**
+ * A Hello World Dto.
+ *
+ * @author Nick Behrens
+ */
+@ApiModel(value = "HelloWorldDto")
+public class HelloWorldDto {
+
+  @ApiModelProperty(required = true, value = "name of message author")
+  private String name;
+  @ApiModelProperty(required = true, value = "message content")
+  private String helloWorldMessage;
+
+  private HelloWorldDto(String name, String message) {
+    this.name = Preconditions.checkNotNull(name);
+    this.helloWorldMessage = Preconditions.checkNotNull(message);
+  }
+
+  /**
+   * Creates a new {@link HelloWorldDto} instance.
+   *
+   * @param name author to be associated with the message
+   * @param message the message to be returned
+   * @return {@link HelloWorldDto} encapsulating a name and message
+   */
+  @JsonCreator
+  public static HelloWorldDto create(@JsonProperty("name") String name, @JsonProperty("hello_world_message") String message) {
```

Comment:
Why is the message using snake case?

Reply:
Because that's what beadledom generated when I ran the `beadledom new` command. I changed it to use camel: https://github.com/cerner/beadledom/pull/149/commits/f8f68c08bbc29dbb7ca87ef6c3e25de9dd562f23